ingestion_tools/scripts/importers/visualization_config.py

```diff
@@ -85,6 +86,7 @@ def _to_segmentation_mask_layer(
             color=color,
             scale=resolution,
             is_visible=file_metadata.get("is_visualization_default"),
+            visible_segments=visible_segments,
```

Comment:
The underlying generate_segmentation_mask_layer function also needs an update. This is related to colors. Currently it only takes a fixed color.